openapi/paths/customers@{id}@edd-search-results@{searchResultId}.yaml

```
@@ -0,0 +1,33 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: searchResultId
+    in: path
+    description: EDD search result ID.
+    required: true
+    schema:
+      type: string
+get:
+  x-products:
+    - Full
+    - Risk Management
+    - Core
+  tags:
+    - Customers
+  summary: Retrieve a customer's EDD search result.
+  operationId: GetEddSearchResult
+  x-sdk-operation-name: GetEddSearchResult
```

I believe that the naming convention for `x-sdk-operation-name` is camelCase?

```suggestion
  x-sdk-operation-name: getEddSearchResult
```

---

I also found another one with PascalCase instead of camelCase [here](https://github.com/Rebilly/api-definitions/blob/main/openapi/paths/customers%40%7Bid%7D%40edd-search-results.yaml#L12).
`GetEddSearchResults` instead `getEddSearchResults`.